src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -37,6 +48,12 @@
   private final static Logger LOGGER = 
         Logger.getLogger(BusinessesService.class.getName());
   private final int ALLOWED_SEARCH_REQUESTS = 3;
+  private Listing currentBusiness;
+  private Iterator<Listing> businesses;
+  private LatLng latLng;
+  private final int minFollowers = 50000;
+  private final String startIndex = "| ";
+  private final String endIndex = "followers";
```

Comment:
Constants should be ALL CAPS.

Additionally, neither startIndex nor endIndex are actually indices (integers).

Reply:
fixed